opds.py

```diff
@@ -252,13 +254,27 @@ def sort_key(work):
             return key
         return sorted(works, key=sort_key)
 
+    def cover_links(self, work):
+        identifier = work.presentation_edition.primary_identifier
```

Comment:
if work doesn't exist or doesn't have the properties, will the exception get caught down the line?